main_pullrequest.go

```diff
@@ -199,9 +199,16 @@ func processGitHubPullRequest(
 
 	// do not start the builds, inform the user about the `start client pipeline` command instead
 	if len(builds) > 0 {
-		// Only comment, if not already commented on a PR
-		botCommentString := ", Let me know if you want to start the client pipeline by " +
-			"mentioning me and the command \""
+		// Two possible pipelines: client or integration
+		var botCommentString string
+		if pr.GetRepo().GetName() == "integration" {
+			botCommentString = `, start a full integration test pipeline with:
+   - mentioning me and ` + "`" + `start integration pipeline` + "`" + ``
+		} else {
+			botCommentString = `, start a full client pipeline with:
+   - mentioning me and ` + "`" + commandStartClientPipeline + `"`
+		}
```

Comment:
Side-note: Maybe we could refactor the comment rendering using `text/template`? It's quite difficult to see how the final string is gonna end up reading this function.

Reply:
I will! Thanks